sentry/src/routes/channel.rs

```diff
@@ -384,6 +489,67 @@ pub async fn add_spender_leaf<C: Locked + 'static>(
     Ok(success_response(serde_json::to_string(&res)?))
 }
 
+/// POST `/v5/channel/0xXXX.../spender/0xXXX...` request
+///
+/// Internally to make the validator worker add a spender leaf in `NewState` we'll just update `Accounting`
+pub async fn add_spender_leaf_axum<C: Locked + 'static>(
+    Extension(app): Extension<Arc<Application<C>>>,
+    Extension(channel): Extension<ChainOf<Channel>>,
+    Path(params): Path<HashMap<String, String>>,
```

Comment:
This should not be a HashMap, use the structs directly, it will use serde::Deserialize for them `Path<(ChannelId, Address>`